setup.py

```diff
@@ -142,7 +142,7 @@ def recursive_data_files(directory, install_directory):
 ]
 
 docs_requires = {
-    "canonical-sphinx",
+    "canonical-sphinx[full]",
```

Comment:
```suggestion
    "canonical-sphinx>=0.2.0[full]",
```

Take it or leave it for now, but when we move to `uv` we'll need to specify minimum versions :-)